server/routes/api/__test__/auth.unit.test.js

```diff
@@ -48,3 +48,34 @@ describe('POST Auth/Login', () => {
     expect(res.body.email).toEqual(email); // returned email should be correct user
   });
 });
+
+describe('POST Auth/Forgot', () => {
+  const existingUserEmail = 'testUser@test.com';
+
+  beforeAll(() => User.create({ email: existingUserEmail, password: 'test' }));
+
+  afterAll(() => User.findOneAndDelete({ email: existingUserEmail }));
```

Comment:
Minor formatting, but it usually is more readable with some spacing between these methods. Can you add a line break after this?